Data-Science/MLflow/bike-sharing-mlflow.ipynb

```diff
@@ -80,6 +57,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "\"#!pip3 install --proxy <PROXY> pydotplus graphviz \\n\",\n",
     "!pip3 install pydotplus graphviz"
```

Comment:
We have also seen that `seaborn` should be installed. It might be not reproducible in DataScience notebook, but it happens in some other notebook.